### DISCUSSION TITLE
Density problems on F7 Lightning Plus

### DISCUSSION THREAD

**Discussion by Jujan** [keirf/greaseweazle#234] [Q&A]
Hello,
I am owner of GW F7 Lightning Plus and have problems with it. I am looking for answers on technical questions.

I have the latest firmware and mostly using FluxMyFluffyFloppy. I am trying to write ADF files for Amiga on 3.5" HD drive and noticed a problem.
If I try to write on HD floppy disks (without density hole covered), then many verification errors comes up and eventually fails.
If I try to write on HD floppy disks (with density hole covered), then many verification errors comes up and eventually fails.
If I try to write on DD floppy disks, then It works just fine without any verification errors.

I have tried other floppy drive and other floppy disks all with the same results. The drives are working normally on HD mode in Windows PC. I can write and read just fine.
It cant be a problem with GW, because it works with DD floppies. It cant be a problem with drives, because I have changed them and works normally on Windows. It cant be problem with media, as DD works and I have tried many new HD disks.

So, where can be a problem?

I am going to elaborate more.
I guess it has something to do with magetic flux level. DD is made from iron oxide and HD is made from cobalt. HD has magnetic coercivity of 700-720 and DD has magnetic coercivity of 600-670. This means the disk gets written on wrong magnetic flux level which would explain why the disk has verification errors, but still working on DD disks just fine.

My goal is to write a DD image on HD disk with correct magnetic flux level, so the data on the HD disks not detoriate. Covering the hole is not a solution to the data detoriation problem. I want to solve this once and for all.

**Comment by keirf** [comment]
DD and HD coercivity on 3.5 inch media is not so different. It's generally accepted that writing to HD media with DD data -- if the drive *thinks* it is DD media -- works okay. The main effect of the DD/HD switch is to adjust drive timing expectations on read.

I can tell you absolutely that writing to HD media with the density hole covered works well for me across multiple drives and disks.

How are you covering the holes? Are you using thick enough tape, or filling the holes? If the covering is weak the switch pressure could depress it and make it look high density still?

**Reply by keirf** [reply]
And by "works okay" I mean some people will argue over longevity of the write, but in short term everyone basically agrees it works. I also happen to think that it's okay longevity-wise too, but that is somewhat hard to prove!

**Reply by Jujan** [reply]
I have covered the hole with black electrical tape. Maybe it really is not enough. I will try with filled hole.
Does GW respect pin 2 /REDWC ?

**Reply by keirf** [reply]
Pin 2 is an input to the drive. Actually on PC 3.5 drives pin 2 is not connected to anything at all.
